hack/e2e.sh

```diff
@@ -21,3 +22,17 @@ if [ -z "$SUITE" ] || [ "$SUITE" == "clusterreader" ]; then
   (awk '/^  azureClusterReaderKubeconfig:/ { print $2 }' <_data/containerservice.yaml | base64 -d) > _data/_out/azure-cluster-reader.kubeconfig
   go test ./test/e2e -test.v -ginkgo.v -ginkgo.focus="\[AzureClusterReader\]" -ginkgo.noColor -ginkgo.randomizeAllSpecs -tags e2e -kubeconfig ../../_data/_out/azure-cluster-reader.kubeconfig 2>&1 | tee azure-reader.log
 fi
+
+if [ -z "$SUITE" ] || [ "$SUITE" == "dedicated-cluster-admin" ]; then
```

Comment:
The rest of the script has been updated to use `[[ ]]`. Please use that here too for consistency.